dev/source/docs/copter-commands-in-guided-mode.rst

```diff
@@ -288,12 +288,12 @@ The message definition can be found `here <https://mavlink.io/en/messages/common
 
 Valid options are:
 
-- MAV_FRAME_GLOBAL (0): alt is meters above sea level
-- MAV_FRAME_GLOBAL_INT (5): alt is meters above sea level
-- MAV_FRAME_GLOBAL_RELATIVE_ALT (3): alt is meters above home
-- MAV_FRAME_GLOBAL_RELATIVE_ALT_INT (6): alt is meters above home
-- MAV_FRAME_GLOBAL_TERRAIN_ALT (10): alt is meters above terrain
-- MAV_FRAME_GLOBAL_TERRAIN_ALT_INT (11): alt is meters above terrain
+- MAV_FRAME_GLOBAL (0): alt in meters above sea level
```

Comment:
all should say "altitude is in meters....", rather than existing or your proposal

Reply:
"alt" in this case is actually a field name so I don't think we should change it to "altitude".

Reply:
that is fine also...

Reply:
actually I didn't think that in this particular context `alt` is variable name, not the short form of `altitude`. So if my change does not make sense, feel free to discard it